README.md

```diff
@@ -11,6 +11,12 @@ To install `saucectl` using NPM just run:
 $ npm i -g saucectl
 ```
 
+In case you need to download `saucectl` from a known source, you can use environment variable:
```

Comment:
Should we mention that this needs to be done either at install or first run (if install scripts are disabled) ?

Reply:
Why do we need to disable the install script?

Reply:
This is the use case of some customers, they disable install scripts for security.
In that case, `SAUCECTL_INSTALL_BINARY` should be passed not-only for `npm install saucectl`, but also the first run of `npx saucectl` / `saucectl` depending how you invoke it.

Reply:
I didn't know about this case. Thanks for raising this question. I've updated my PR.